### PR TITLE
agda 2.5.4.2

### PR DESCRIPTION
**Pull request by mietek**
Updates Agda to 2.5.4.2 and the Agda standard library to 0.17.

### CODE REVIEW

File: Formula/agda.rb [hunk 1]
```diff
@@ -7,12 +7,12 @@ class Agda < Formula
...
   stable do
-    url "https://hackage.haskell.org/package/Agda-2.5.4.1/Agda-2.5.4.1.tar.gz"
-    sha256 "7759aa76936e6a35325c2e186a7546553921775155a426c8edc9a234f58ab72f"
+    url "https://hackage.haskell.org/package/Agda-2.5.4.2/Agda-2.5.4.2.tar.gz"
+    sha256 "f645add8a47a35da3f37757204fa9c80aeb0993d628fc2057fa343e92e579b1f"
 
     resource "stdlib" do
       url "https://github.com/agda/agda-stdlib.git",
-          :revision => "a1a10b39d35b8fc40e87723a89f5682252d46380"
+          :tag => "v0.17"
```
Comment (commitay) [inline Formula/agda.rb]: This should be pinned to a revision, not a tag.

Comment (mietek) [reply Formula/agda.rb]: Should that be part of `brew audit --strict`, then?

Comment (commitay) [reply Formula/agda.rb]: Yes, it should actually be a normal `brew audit` but it isn't yet.

**Comment by mietek** [comment]
All checks have passed.

**Pull request closed without merging**